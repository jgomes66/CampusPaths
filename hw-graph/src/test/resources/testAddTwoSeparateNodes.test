# Test Adding Two Separate Nodes to Graph
# Testing that more than one node can be added to graph

# Create a graph
CreateGraph newGraph

# Add two nodes to graph
AddNode newGraph firstNode
AddNode newGraph secondNode

# Check if graph contains both nodes
ListNodes newGraph