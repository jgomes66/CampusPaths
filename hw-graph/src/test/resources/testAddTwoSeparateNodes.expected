# Test Adding Two Separate Nodes to Graph
# Testing that more than one node can be added to graph

# Create a graph
created graph newGraph

# Add two nodes to graph
added firstNode to newGraph
added secondNode to newGraph

# Check if graph contains both nodes
newGraph contains: firstNode secondNode